assembl/static2/js/app/components/common/nuggetsManager.js

```diff
@@ -0,0 +1,43 @@
+// @flow
+
+import Nuggets from '../debate/thread/nuggets';
+
+const invalidIndex = (index) => {
+  return index === -1;
```

Comment:
actually, an invalid index is `index < 0`. But, I understand this case will never be hit in this PR.

Reply:
from MDN
>Array.prototype.indexOf()
>....
>Return value
>The first index of the element in the array; -1 if not found.

Reply:
I understand. The method name implies it checks all cases of invalid indices, not just the returned value of `indexOf`. Perhaps change the method name?

Reply:
yup